fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_createErrorMsg.vail

```diff
@@ -0,0 +1,48 @@
+// Create a FHIR OperationOutcome message from an error.
```

Comment:
Unless you have a desire for self torture, you'll have to trust me on the FHIR message contents.  There's an example in the test so you can see why the vast majority of time is spent sorting those out.